test_package/conanfile.py

```diff
@@ -7,6 +7,8 @@ class TestPackageConan(ConanFile):
     generators = "cmake"
 
     def build(self):
+        if self.settings.os == "Linux":
+            self.settings.compiler.libcxx = "libstdc++11"
```

Comment:
why is this necessary ?
Could we instead raise ConanInvalidConfiguration in the main recipe if libcxx is not supported ?

Reply:
I'm actually very not sure about these two lines. If I don't specify `libstdc++11` here then the test package won't link with `undefined reference to` std stuff that is usually libstd incompatibility. I assume that `conan_basic_setup` in the test's `CMakeFile.txt` does something that is not done in the `.gn` build files of `crashpad` and `mini_chromium`.

I couldn't find the right way to tell the test package to use `libstdc++11`.

In the main recipe, there's an explicit `del self.settings.compiler.libcxx` in the `configure` function that I don't get either.

So you're suggesting to throw in `configure` like

```py
if self.settings.os == "Linux" and self.settings.compiler.libcxx != "libstdc++11":
   raise ConanInvalidConfiguration()
```
?

Reply:
to be short: yes !